adding ethics-committee-with-text image (#3377)
the-turing-way/the-turing-way: book/website/ethical-research/ethics-committees.md

This is not to liberate the researcher of their ethical responsibilities -- ultimately, they make the decisions as to how to conduct their work -- but to provide support in respect to these questions.
Whilst there is a great deal of complexity involved in holding individuals accountable in scientific projects, this chapter assumes that RECs seek to improve how science is conducted at large.

> ```{figure}../../figures/ethics-committee-with-text.*

Reviewer: ```suggestion
```{figure} ../../figures/ethics-committee-with-text.*
```

> ```{figure} ../../figures/ethics-committee-with-text.*

Reviewer: ```suggestion
```{figure}../../figures/ethics-committee-with-text.*
```

> ```{figure} ../../figures/ethics-committee-with-text.*

Reviewer: ```suggestion
```{figure} ../figures/ethics-committee-with-text.*
```

---
height: 500px
name: ethics-committee-with-text

> alt: A female presenting person is walking into a path that leads to somewhere far with mountains. The path starts with a big banner that says “Research Ethics Committee”. At the beginning of the path, there is a male presenting person with a map, a backpack and a friendly face looking like they are ready to join the other person to embark on the research ethics committee journey. Past the first part of the path, there are a couple of booths, one with “Ethics Reviewers”, and another one further down with “ethics decision making” as the title of the booth. These represent the journey one goes through when getting into the research ethics committee processes.

Reviewer 2: ```suggestion
alt: A female-presenting person is walking into a path that leads to somewhere far with mountains. The path starts with a big banner that says “Research Ethics Committee”. At the beginning of the path, there is a male-presenting person with a map, a backpack, glasses and a friendly face looking like they are ready to join the other person to embark on their research ethics committee journey. Next to the first part of the path, there are a couple of booths full of people, one with “Ethics Reviewers”, and another one further down with “Ethics Decision Making” as the title of the booth. These represent the journey one goes through when going through research ethics committee processes.
```
